flink-baseline/flink-common/src/main/java/com/ness/flink/assigner/WindowGeneratorWatermarkWithIdle.java

```diff
@@ -1,18 +1,16 @@
-/*
- * Copyright 2020-2023 Ness USA, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+//Copyright 2021-2023 Ness Digital Engineering
```

Comment:
Please return back formatting to way how it was before with using
```
/*
 * Copyright 2021-2023 Ness Digital Engineering
 *
 *
 *  
 */
```

You should only adjust TEXT not Format !

Reply:
Done